### ISSUE TITLE
Error in dlopen or dlsym: libthnvrtc.so

### ISSUE THREAD

**Issue by februa22** [NVIDIA/tensorrt-inference-server#1024]
**Description**
The TRTIS server has a PyTorch model. When it gets an inference request it terminated called after throwing an instance of `std::runtime_error`. The following is its log message:

```sh
===============================
== TensorRT Inference Server ==
===============================

NVIDIA Release 19.06 (build 6791108)

Copyright (c) 2018-2019, NVIDIA CORPORATION.  All rights reserved.
Copyright 2019 The TensorFlow Authors.  All rights reserved.
Copyright 2019 The TensorFlow Serving Authors.  All rights reserved.
Copyright (c) 2016-present, Facebook Inc. All rights reserved.

Various files include modifications (c) NVIDIA CORPORATION.  All rights reserved.
NVIDIA modifications are covered by the license terms that apply to the underlying
project or file.

I0114 00:37:32.995674 1 main.cc:325] Starting endpoints, 'inference:0' listening on
I0114 00:37:32.995959 1 grpc_server.cc:263] Starting a GRPCService at 0.0.0.0:8001
I0114 00:37:32.995977 1 grpc_server.cc:269] Register TensorRT GRPCService
I0114 00:37:32.996012 1 grpc_server.cc:272] Register Infer RPC
I0114 00:37:32.996020 1 grpc_server.cc:276] Register StreamInfer RPC
I0114 00:37:32.996027 1 grpc_server.cc:281] Register Status RPC
I0114 00:37:32.996033 1 grpc_server.cc:285] Register Profile RPC
I0114 00:37:32.996042 1 grpc_server.cc:289] Register Health RPC
I0114 00:37:32.996048 1 grpc_server.cc:301] Register Executor
I0114 00:37:33.005743 1 http_server.cc:579] Starting HTTPService at 0.0.0.0:8000
I0114 00:37:33.051511 1 main.cc:356]  localhost:8002 for metric reporting
I0114 00:37:33.054752 1 metrics.cc:149] found 1 GPUs supporting NVML metrics
I0114 00:37:33.061957 1 metrics.cc:158]   GPU 0: TITAN Xp
I0114 00:37:33.062977 1 server.cc:113] Initializing TensorRT Inference Server
I0114 00:37:33.123575 1 server_status.cc:83] New status tracking for model 'ctdet'
I0114 00:37:33.123684 1 model_repository_manager.cc:627] loading: ctdet:1
I0114 00:37:33.126241 1 libtorch_backend.cc:189] Creating instance ctdet_0_0_gpu0 on GPU 0 (6.1) using model.pt
I0114 00:37:38.858033 1 model_repository_manager.cc:758] successfully loaded 'ctdet' version 1
terminate called after throwing an instance of 'std::runtime_error'
  what():  

error in dlopen or dlsym: libthnvrtc.so: cannot open shared object file: No such file or directory:
operation failed in interpreter:
:
operation failed in interpreter:
```

And the following is the error message of the Python client API:

```python
/usr/local/lib/python3.5/dist-packages/tensorrtserver/api/__init__.py:876: in run
    self._last_request_id = _raise_if_error(c_void_p(_crequest_infer_ctx_run(self._ctx)))
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

err = c_void_p(31019024)

    def _raise_if_error(err):
        """
        Raise InferenceServerException if 'err' is non-success.
        Otherwise return the request ID.
        """
        if err.value is not None:
            ex = InferenceServerException(err)
            isok = _crequest_error_isok(err)
            _crequest_error_del(err)
            if not isok:
>               raise ex
E               tensorrtserver.api.InferenceServerException: [ 0] GRPC client failed: 14: Socket closed
```

**TRTIS Information**
I'm using the TRTIS container `nvcr.io/nvidia/tensorrtserver:19.06-py3`

**To Reproduce**

- Run an inference server using the TRTIS container: `docker run -dt --gpus '"device=<gpu_id>"' --shm-size=1g --ulimit memlock=-1 --ulimit stack=67108864 -p8000:8000 -p8001:8001 -p8002:8002 -v/path/to/model/repository:/models nvcr.io/nvidia/tensorrtserver:19.06-py3 trtserver --model-store=/models`
- Send an inference request to the server with Python client API

**Expected behavior**
The client gets an inference result from the server successfully.

**In addition**
I'm using TRTIS `19.06` version because I need the PyTorch `1.1.0`.
If possible, I would like to use the TRTIS `19.10` with the PyTorch `1.1.0`.
Is there any way to build it like that?

Thanks in advance.

**Comment by CoderHam** [comment]
I believe there were small API changes between Pytorch 1.1.0 to 1.2.0. If you can re-work those changes in the **libtorch_backend.cc** you can then uninstall the current version of pytorch in the build container and re-build pytorch 1.1.0. This should technically work. You can then use this build-container to build your final container. 

A few things to note: 

- If possible try to use Pytorch 1.3.0 as is the version in our latest release 1.3.0
- There were a few fixes made to the pytorch backend in 19.07 and 19.08

**Comment by februa22** [comment]
@CoderHam 
I decided to use TRTIS 19.10 with Pytorch 1.3.0.
Thank you very much! 👍
